Clicker/Views/PollBuilder/MCPollBuilderView.swift

```diff
@@ -137,20 +140,30 @@ class MCPollBuilderView: UIView {
             opt.totalOptions = mcOptionModels.count - 1
         }
     }
+
+    func shouldLightenDraftsText(_ shouldLighten: Bool) {
+        guard let draftsTitleModel = draftsTitleModel,
+            let sectionController = adapter.sectionController(for: draftsTitleModel) as? DraftsTitleSectionController else {
+            return
+        }
+        sectionController.shouldLightenText(shouldLighten)
+    }
     
     // MARK: - KEYBOARD
     @objc func keyboardWillShow(notification: NSNotification) {
         if let _ = (notification.userInfo?[UIKeyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue {
             let contentInsets = UIEdgeInsetsMake(0.0, 0.0, 70, 0.0)
             collectionView.contentInset = contentInsets
             collectionView.superview?.layoutIfNeeded()
```

Comment:
why are you doing this?

Reply:
When the keyboard comes up, we need to shift the content inset up a bit so that we can see the multiple choice option